sdk/ai/Azure.AI.Inference/CHANGELOG.md

```diff
@@ -10,6 +10,8 @@
 
 ### Bugs Fixed
 
+- Fixed an issue with the audio samples not properly showcasing the intended workflow.
+
 ### Other Changes
```

Comment:
I would remove any empty section headers

Reply:
I always forget to do that....